grc/gui/ParamWidgets.py

```diff
@@ -361,7 +361,7 @@ def _handle_clicked(self, widget=None):
         dir_dialog.set_local_only(True)
         dir_dialog.set_select_multiple(False)
         
-        # Show dialog and update paramter on success
+        # Show dialog and update parame:wqter on success
```

Comment:
this one doesn't look right

Reply:
:facepalm:  
will fix..thanks

Reply:
Using vim sucks sometimes :neutral_face:  
Fixed in 2ff5af5